docs/core/entity/sensor.md

```diff
@@ -29,6 +29,7 @@ If specifying a device class, your sensor entity will need to also return the co
 | carbon_dioxide | ppm | parts per million of carbon dioxide concentration
 | carbon_monoxide | ppm | parts per million of carbon monoxide concentration
 | humidity | % | % of humidity in the air.
+| gas | m³/ft³ | Amount of gas.
```

Comment:
Volume of gas (according to unit) and amount of gas (by description) are not the same thing as gas is a compressible medium. The description is probably better as volume since this relates to energy. This device class would not work for say a CO2 level sensor in a room